dockstore-integration-testing/src/test/java/io/dockstore/client/cli/DAGWorkflowTestIT.java

```diff
@@ -300,19 +302,29 @@ public void testGetStepsArrayInsteadOfObject() throws ApiException {
     /**
      * This tests that a WDL workflow with complex imports is properly imported (also tests absolute paths)
      *
+     * Note: With fix for https://github.com/dockstore/dockstore/issues/2931, this WDL now fails. Adjusting test accordingly. If you run
```

Comment:
In the original spirit of the test, it seems to me that we should create a new test with a complex workflow that does validate. Perhaps even the workflow added in this PR?

Reply:
Added tests to WDLHandlerTest for DAG in upcoming push. Preferred to make them unit(ish) tests instead of integration tests, because I don't want to commit the huge workflow to DockstoreTestUser2 and slow down all tests.